types/alloy/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "dom",
+            "es6"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": false,
```

Comment:
[These should all be set to true](https://github.com/definitelytyped/definitelytyped#some-packages-have-no-tslintjson-and-some-tsconfigjson-are-missing-noimplicitany-true-noimplicitthis-true-or-strictnullchecks-true)

Reply:
Got it - copy-pasted from an existing definition. That was probably a mistake, as I can see now from the docs. Sorry!